### DISCUSSION TITLE
Openai assistant

### DISCUSSION THREAD

**Discussion by valepetros** [LiveHelperChat/livehelperchat#2148] [Q&A]
I'm using a chatbot follwing the guidelines of the tutorial and it works great!

Now I'm trying to use the openai API for the chatgpt assistant.
In the openai panel I've created an assistant and the API works well using postman.
If I use the same postman configuration in live helper chat it doeasn't work.

Is it possible that I'm configuring something wrong or is it not possible to use a chatgpt assistant in live helper chat?

this is the postman cUrl

```
curl --location 'https://api.openai.com/v1/assistants/asst_XXX' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer sk-proj-XXX' \
--header 'OpenAI-Beta: assistants=v2' \
--header 'Cookie: __cf_bm=bUfKJhSyB9ZfYuTlKsi0RI3SFt49jkTh87jISoqlRas-1728067983-1.0.1.1-xVl2ZVcKAQa.iT1bYl0Q.VzM1RRsscRtO2FwnVAlBWiANxLtvEhtYfG1g.jAOIoj0m0u5SSNn7NKaP54M9MUxA; _cfuvid=mdWZvNHvJc.iE9kDcJ74vHMeW.fcIq61lm69Gm64OZM-1728066915101-0.0.1.1-604800000' \
--data '{
  "instructions": "You are a helpful assistant."
}'
```

In the LHC panel of the chatbot i've used these configuration:

host: https://api.openai.com
method: POST
Sub URL: /v1/assistants/asst_XXX
Authorization method: Bearer token sk-proj-XXX
Headers: OpenAI-Beta : assistants=v2, Content-Type: application/json,
Body(raw Json):
{
  "instructions": "You are a helpful assistant."
}

Thanks

**Comment by remdex** [comment]
Please watch youtube video I did and extension i prepared just for openai integration...


https://github.com/LiveHelperChat/chatGPT

**Reply by valepetros** [reply]
> Please watch youtube video I did and extension i prepared just for openai integration...
> 
> https://github.com/LiveHelperChat/chatGPT

Thank you!

The ChatGPT module is available in the latest version only or can I add only the module in my version (4.31)?

**Reply by remdex** [reply]
Can't you see a link?

It's supported only in latest version because of polling method implementation in new version. So just download code from from master branch.

**Reply by valepetros** [reply]
> Can't you see a link?
> 
> It's supported only in latest version because of polling method implementation in new version. So just download code from from master branch.

I've installed and configured the chatgpt module and works GREAT!

Thank you so much!
